packages/modul-components/src/components/phonefield/phonefield.ts

```diff
@@ -82,6 +88,21 @@ export class MPhonefield extends ModulVue {
     i18nCountryLabel: string = this.$i18n.translate('m-phonefield:country-label');
     i18nExample: string = this.$i18n.translate('m-phonefield:example');
 
+    beforeMount(): void {
+        // sprites-flags.svg is a very big file, this is why sprites should only be added to the DOM before this component is mounted.
+        const sprites: string = require('../../assets/icons/sprites-flags.svg');
+        const svg: SpritesService = this.$svg;
+        if (this.externalSprite) {
+            if (svg.isInExternalSprites('mflag')) {
+                svg.addExternalSprites(sprites, 'mflag');
+            }
+        } else {
+            if (!document.getElementById('mflag-svg__flag-ae')) {
+                svg.addInternalSprites(sprites);
+            }
+        }
```

Comment:
Il n'y a pas un bug ici? ça serait pas mieux de faire ça?
if(this.externalSprite && !svg.isInExternalSprites('mflag')) {
svg.addExternalSprites(sprites, 'mflag')
}

Reply:
effectivement c'est bien bien `!svg.isInExternalSprites` 👍